conditional/templates/errors.html

```diff
@@ -1,8 +1,10 @@
 {% extends "nav.html" %}
 {% block title %}Error {{ error_code }}{% endblock %}
 {% block body %}
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 <div class="container main error-page align-center">
     <div class="col-xs-12">
+
```

Comment:
Was the newline added on purpose?

Reply:
No, I just didn't remove it. That was where I tried putting the <meta> tag first